src/pkg/projects/Microsoft.NETCore.DotNetHost/host.wxs

```diff
@@ -7,17 +7,36 @@
 
     <ComponentGroup Id="InstallSharedHostandDetectionKeys">
 
+      <!-- When installing the SharedHost; copy all files to the traditional default install location: 'ProgramFiles'\dotnet even when installing into a custom location.
+
+      User Scenerio:
+      When a user has installed a 2.1 SDK or similar SDK first so that the PATH environmental variable has 'ProgramFiles'\dotnet in a first position.
+      Then the user installs a 3.0 SDK thru VS to a different volume with VS so that 'VSSharedVolume'\dotnet is in a second position.
+      We want the 3.0 SharedHost to be on the PATH in the first position. Therefore, installing the SharedHost to both 'VSSharedVolume'\dotnet and
+      'ProgramFiles'\dotnet will achieve this.
+
+      On a 'clean' machine when installing into a custom location; the installation behavior is the same.
+      Rationale:
+        1. the legacy SDK can be subsequently installed
+        2. the user runs 'dotnet' commands directly against 'ProgramFiles'\dotnet\dotnet.exe -->
```

Comment:
Can you wrap this to something more modest? I go with 100.

On the other hand, I feel like we'd want this in a markdown doc (unfortunately don't know where 😕) so it's easier to point devs confused by the behavior to this. Maybe there's a dotnet/designs doc we can link to from a code comment, something like that?

Reply:
The description looks good, but I agree with @dagood that we should document this somewhere we can point external people to- source code isn't really it.

Reply:
Filed https://github.com/dotnet/core-setup/issues/7737 to follow up on this.